city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,129 @@
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthernIlUniversitySpider(CityScrapersSpider):
+    name = "chi_northern_il_university"
+    agency = "Northeastern Illinois University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.neiu.edu/about/board-of-trustees/board-meeting-materials"]
+
+    def parse(self, response):
+        self.link_date_map = defaultdict(list)
+        link_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]/h3'
+        )
+        for title in link_list:
+            full_str = title.xpath(".//text()").extract()[0]
+            date = full_str.split("-")[0].split("(")[0]
+
+            date_obj = datetime.strptime(date.strip(" "), '%B %d, %Y')
+            new_date_obj = datetime(date_obj.year, date_obj.month, date_obj.day, 13, 00)
+            for one_day_links in title.xpath('.//following-sibling::ul[1]/li'):
+                href = one_day_links.xpath('.//@href').get()
+                name = one_day_links.xpath('.//text()').extract()
+                if href is None:
+                    link = ''
+                else:
+                    link = response.urljoin(href)
+                self.link_date_map.setdefault(new_date_obj, []).append({
+                    "title": ''.join(name),
+                    "href": link
+                })
+        yield response.follow(
+            "https://www.neiu.edu/about/board-of-trustees/calendar-of-meetings",
+            callback=self._parse_detail
+        )
+
+    def _parse_detail(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        items_list = response.xpath(
+            '//div[@class="field field--name-field-generic-body field--type-text-long \
+field--label-hidden field--item"]'
+        )
+        year_list = items_list.xpath('.//h2')
+        for years in year_list.extract():
+            year = re.findall(r'\d{4}', years)[0]
+            items = year_list.xpath('.//following-sibling::ul[1]/li').extract()
+            for item in items:
+                start = self._parse_start(item, year)
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    description='',
+                    classification=self._parse_classification(item),
+                    start=start,
+                    end=None,
+                    all_day=False,
+                    time_notes="See agenda for meeting time",
+                    location=self._parse_location(item),
+                    links=self.link_date_map[start],
+                    source=self._parse_source(response),
+                )
+                meeting["status"] = self._get_status(meeting)
```

Comment:
We should pass the full text of the `li` item to this as a kwarg since it will set the status to `CANCELLED` if it sees any mention of cancellation in the text